app/views/submissions/_task.html.haml

```diff
@@ -0,0 +1,34 @@
+%section.ccs-task{id: "task-#{task.id}"}
+  %header.ccs-task__information
+    .ccs-task__statuses
+      - if task.complete?
+        %strong.govuk-tag.govuk-tag__success Completed
+      - if task.errors?
+        %strong.govuk-tag.govuk-tag__failure Errors
+      - if task.late?
+        %strong.govuk-tag.govuk-tag__warning Late
+    .ccs-task__body
+      - if current_user.multiple_suppliers?
+        %h5.govuk-body
+          = task.supplier_name
+      %h3.govuk-heading-m
+        = task.framework.short_name
+        = task.framework.name
+      %h4.govuk-heading-s
+        Report management information for
+        = task.period_month_name
+      - unless task.complete?
+        %p
+          Due
+          = task_due_date(task)
+  .ccs-task__action
+    .ccs-task__action--primary
+      - case task.status
```

Comment:
We're using `case` here for just two options, and 3 `if`s at line 4. Would be good to stick to one or the other.

Reply:
I think this is by design. A task might be 'late' and in 'error'?

Reply:
ah, that makes sense!